protocols/Imperial College/README.md

```diff
@@ -0,0 +1,38 @@
+# Drug Mixing and Transferring
```

Comment:
Please avoid spaces in the directory names - I think most of our folders look like `ImperialCollege/` or maybe `Imperial_College/`. The reason for no spaces is that the folder name becomes the URL for the web page, and having spaces in URLs can make link sharing not work right. Eg:

http://staging-a.protocols.opentrons.com/protocol/Imperial College 

is a broken link b/c of the space